cloud/aws/modules/ecs_fargate_service/main.tf

```diff
@@ -0,0 +1,312 @@
+# S3 BUCKET - For access logs
+#------------------------------------------------------------------------------
+data "aws_elb_service_account" "default" {}
+
+locals {
+  # While we don't really need to do this, it keeps resource names consistent
+  # with previous deploys to avoid destroying and recreating a bunch of
+  # resources.
+  name_prefix = "${var.app_prefix}-civiform"
+}
+
+#------------------------------------------------------------------------------
+# APPLICATION LOAD BALANCER
+#------------------------------------------------------------------------------
+resource "aws_lb" "civiform_lb" {
+  name = substr("${local.name_prefix}-lb", 0, 31)
+
+  internal                         = false
+  load_balancer_type               = "application"
+  drop_invalid_header_fields       = false
+  subnets                          = var.public_subnets
+  idle_timeout                     = 60
+  enable_deletion_protection       = false
+  enable_cross_zone_load_balancing = false
+  enable_http2                     = true
+  ip_address_type                  = "ipv4"
+  security_groups                  = [aws_security_group.lb_access_sg.id]
+
+  tags = merge(
+    var.tags,
+    {
+      Name = "${local.name_prefix}-lb"
+    },
+  )
+}
+
+moved {
+  from = module.ecs-alb[0].aws_lb.lb
+  to   = aws_lb.civiform_lb
+}
+
+#------------------------------------------------------------------------------
+# ACCESS CONTROL TO APPLICATION LOAD BALANCER
+#------------------------------------------------------------------------------
+resource "aws_security_group" "lb_access_sg" {
+  name        = "${local.name_prefix}-lb-access-sg"
+  description = "Controls access to the Load Balancer"
+  vpc_id      = var.vpc_id
+
+  egress {
+    protocol    = "-1"
+    from_port   = 0
+    to_port     = 0
+    cidr_blocks = ["0.0.0.0/0"]
+  }
+
+  tags = merge(
+    var.tags,
+    {
+      Name = "${local.name_prefix}-lb-access-sg"
+    },
+  )
+}
+
+moved {
+  from = module.ecs-alb[0].aws_security_group.lb_access_sg
+  to   = aws_security_group.lb_access_sg
+}
+
+resource "aws_security_group_rule" "ingress_through_http" {
+  security_group_id = aws_security_group.lb_access_sg.id
+  type              = "ingress"
+  from_port         = 80
+  to_port           = 80
+  protocol          = "tcp"
+  cidr_blocks       = ["0.0.0.0/0"]
+  prefix_list_ids   = []
+}
+
+moved {
+  from = module.ecs-alb[0].aws_security_group_rule.ingress_through_http["default_http"]
+  to   = aws_security_group_rule.ingress_through_http
+}
+
+resource "aws_security_group_rule" "ingress_through_https" {
+  security_group_id = aws_security_group.lb_access_sg.id
+  type              = "ingress"
+  from_port         = 443
+  to_port           = 443
+  protocol          = "tcp"
+  cidr_blocks       = ["0.0.0.0/0"]
+  prefix_list_ids   = []
+}
+
+moved {
+  from = module.ecs-alb[0].aws_security_group_rule.ingress_through_https["default_http"]
+  to   = aws_security_group_rule.ingress_through_https
+}
```

Comment:
**question (non-blocking):** how long do we need to leave this `moved` statements in? until everyone has deployed with these changes at least once?

Reply:
cool